images/win/Windows2016-Azure.json

```diff
@@ -385,7 +384,8 @@
             "type": "powershell",
             "scripts":[
                 "{{ template_dir }}/scripts/Installers/Configure-Antivirus.ps1",
-                "{{ template_dir }}/scripts/Installers/Disable-JITDebugger.ps1"
+                "{{ template_dir }}/scripts/Installers/Disable-JITDebugger.ps1",
+                "{{ template_dir }}/scripts/Installers/Post-Deployment.ps1"
```

Comment:
I suggest to name this script `Configure-Shell.ps1`. It is pretty big so I don't think that we would like to add something to it.
Also I think both `Configure-Shell.ps1` and `Disable-JITDebugger.ps1` should be run before `Finalize-VM.ps1`